jwst/pipeline/calwebb_image2.py

```diff
@@ -23,7 +23,7 @@ class Image2Pipeline(Pipeline):
     Process JWST imaging-mode slope data from Level-2a to Level-2b.
 
     Included steps are:
-    background_subtraction, assign_wcs, flat_field, photom and resample.
+    background, assign_wcs, flat_field, photom and resample.
```

Comment:
I think for docstring purposes, it would be most intuitive if this was the step's alias used in the pipeline, i.e. ``bkg_subtract``.

Reply:
Just here - for users looking at the Image2Pipeline docstring, they may want to access its steps, and while the other steps are listed here by their step alias used in the pipeline code, the entry for background subtraction does not.